supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -0,0 +1,69 @@
+[[red-hat-enterprise-linux-conventions]]
+
+For documentation questions, contact rhel-docs@redhat.com.
+
+[discrete]
+[[clean-install]]
+==== clean install (noun)
+*Description*: A clean install removes all traces of the previously installed operating system, system data, configurations, and applications and installs the latest version of the operating system.
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: xref:upgrade[upgrade], xref:in-place-upgrade[in-place upgrade]
```

Comment:
Any of these xrefs will need to be updated to use the new anchor ID (because they currently go to the existing "upgrade" term in the general term section.

Reply:
Fixed